types/react/index.d.ts

```diff
@@ -320,7 +321,7 @@ declare namespace React {
         /**
          * **NOTE**: Exotic components are not callable.
          */
-        (props: P): (ReactElement|null);
```

Comment:
This seems unrelated. Probably branched off from another branch instead of `master`?

Reply:
no it is from master only

Reply:
Fixed, it mistakenly got cut off from a different branch